README.md

```diff
@@ -33,6 +33,21 @@ Starter Name | Version for Spring Boot 2.1.x | Version for Spring Boot 2.0.x
 [spring-data-gremlin-boot-starter](azure-spring-boot-starters/spring-data-gremlin-boot-starter/README.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/spring-data-gremlin-boot-starter.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.microsoft.azure%22%20AND%20a%3A%22spring-data-gremlin-boot-starter%22) | [![](https://img.shields.io/maven-central/v/com.microsoft.azure/spring-data-gremlin-boot-starter/2.0.svg)](https://search.maven.org/artifact/com.microsoft.azure/spring-data-gremlin-boot-starter/2.0.8/jar)
 [azure-spring-boot-metrics-starter](azure-spring-boot-starters/azure-spring-boot-metrics-starter/README.md) | [![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-spring-boot-metrics-starter.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.microsoft.azure%22%20AND%20a%3A%22azure-spring-boot-metrics-starter%22) | [![](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-spring-boot-metrics-starter/2.0.svg)](https://search.maven.org/artifact/com.microsoft.azure/azure-spring-boot-metrics-starter/2.0.8/jar)
 
+### Snapshots

```

Comment:
Add links for the snapshot packages in the table of `usage` section? So that users know what is the latest snapshot version.

Reply:
Make sense, but I prefer to add the links to Snapshots only instead of usage, thinking to provide the same snapshot version for the whole repo.